service_configuration_lib/spark_config.py

```diff
@@ -791,6 +831,7 @@ def get_spark_conf(
     load_paasta_default_volumes: bool = False,
     aws_region: Optional[str] = None,
     service_account_name: Optional[str] = None,
+    force_spark_resource_configs: bool = True,
```

Comment:
Setting `force_spark_resource_configs` default to be True at this moment. Will create another change to set it to False as default after both `paasta` and `spark-tools` are updated and support `--force-spark-resource-configs` option.

Reply:
Good point.

Reply:
No need to add this option here as this method is only used for mesos. On that note, we should also create a ticket to deprecate all the mesos logic from service_configuration_lib, spark_tools and paasta spark_run

Reply:
=> This is for `get_spark_conf()` function